src/main/java/edu/cornell/kfs/fp/service/RecurringDisbursementVoucherDocumentService.java

```diff
@@ -58,12 +58,19 @@
 	 * @return
 	 */
 	Collection<PaymentDetail> findPaymentDetailsFromRecurringDisbursementVoucher(RecurringDisbursementVoucherDocument recurringDisbursementVoucherDocument);
-	
+
 	/**
 	 * Determines if the payment can be canceled from the Recurring DV by the user.
 	 * @param user
 	 * @param paymentDetailStatus
 	 * @return True if the payment can be canceled.  False if ii can't be.
 	 */
 	boolean isPaymentCancelable(Person user, String paymentDetailStatus);
+
+	/**
+	 * This function should be called by a batch process.
+	 * Find all open DV spawned by a Recurring DV, and route and blanket approve those whose due date is today or in the past.
+	 * @return boolean
+	 */
+	boolean autoApproveDisbursementVouchersSpawnedByRecurringDvs();
```

Comment:
The docs are a bit mis leading, it should approve DVs whose due date are the last day of the open fiscal month or earlier